docs/sources/tempo/setup/operator/monitor.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Monitor'
```

Comment:
The single quote isn't needed for the title in front matter. For more information about front matter, see [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/write/front-matter/).
```suggestion
title: Monitor
```